composer.json

```diff
@@ -83,7 +83,7 @@
         "matthiasmullie/minify": "^1.3",
         "monolog/monolog": "^2.0",
         "nelmio/cors-bundle": "^2.0.1",
-        "nelmio/security-bundle": "^2.2",
+        "nelmio/security-bundle": "^2.12 || ^3.0",
```

Comment:
Could we also just use version 3?

Reply:
That would be unnecessarily restrictive imo. We also require a broad range of major versions of other dependencies like `psr/log` or `psr/http-message`. We also allow multiple major versions for Symfony packages (at least in 4.13). Since we only required `^2.2` before and the current state of this PR is fully compatible with `^2.12 || ^3.0` I think we should still allow `^2.12`.